scm/pipeline.go

```diff
@@ -0,0 +1,14 @@
+package scm
+
+import "time"
+
+type (
+	Pipeline struct {
+		ID       string     `json:"id"`
+		Status   string     `json:"status"`
+		Created  time.Time  `json:"created_at"`
+		Updated  *time.Time `json:"updated_at,omitempty"`
```

Comment:
we don't use pointers for any other dates, so I would recommend removing the pointer:

```diff
-		Updated  *time.Time `json:"updated_at,omitempty"`
+		Updated  time.Time
```

if the goal is to determinate if the date is unset, you can use the `IsZero` function to check if the time is the zero value / unset

Reply:
Update to time.Time